### DISCUSSION TITLE
Links stopped working

### DISCUSSION THREAD

**Discussion by defactomail** [YOURLS/YOURLS#3342] [General]
Hi. All of a sudden all of my links stopped working. If I log into YOURLS and copy the original link for a shortened link, it work perfectly. But when I click on a link that has been shortened, it opens a new tab with a blue bar at the top saying "Powered by YOURLS" and nothing happens. It just keeps spinning but won't resolve the URL.

Please help with this. I've got dozens and dozens of links all through out my website that suddenly don't work.

I did restart the server, but that didn't help. I installed into domain.com/shop/, but as I said, it was working fine  a couple of days ago. The only thing I can think of is that I changed the PHP version from lsphp74 to lsphp80 and then back to lsphp74.

**Comment by dgw** [comment]
Sounds like the built-in toolbar plugin is active.

![image](https://user-images.githubusercontent.com/164140/169346395-2d418dd8-2331-45d9-ad7d-1331ab7a6588.png)

If you don't want the bar, deactivate this plugin. Further [troubleshooting](https://docs.yourls.org/guide/troubleshooting/first-steps.html) may be required if shortlinks still do not work.
